src/modules/commander/HealthAndArmingChecks/checks/estimatorCheck.cpp

```diff
@@ -450,6 +450,22 @@ void EstimatorChecks::checkEstimatorStatus(const Context &context, Report &repor
 							    events::ID("check_estimator_gps_vert_speed_drift_too_high"),
 							    log_level, "GPS Vertical Speed Drift too high");
 
+			} else if (estimator_status.gps_check_fail_flags & (1 << estimator_status_s::GPS_CHECK_FAIL_SPOOFED)) {
```

Comment:
Now that we're getting this through the estimator I would delete all the checks directly on vehicle_gps_position spoofing.

The check through the estimator is better because we can decide to ignore it or need through the EKF2_GPS_CHECK bit.

Reply:
The initial thought was that we always have an event with the first spoofed message (as mentioned in the other comment), independent of the automatic disabling of the fusion. This is why I still have both. We need to define which option we want to choose